### DISCUSSION TITLE
Windows 11 latest build fails - 22621.2715

### DISCUSSION THREAD

**Discussion by oznetmaster** [valinet/ExplorerPatcher#2455] [General]
The latest Windows 11 build update (22621.2715) fails with all versions of explorer patch :(

**Comment by pyrates999** [comment]
This is discussed here: https://github.com/valinet/ExplorerPatcher/issues/2457

**Comment by m-brooks** [comment]
I am also suffering this issue. I have no other tweak software installed. I have "Disable context menu" enabled. Weird thing is, I can find nothing in eventvwr. Here is my console trace:

funchook create 1
Running on Windows 11, OS Build 10.0.22621.2715.
Loaded symbols
Setup ntdll functions done
Setup user32 functions done
Patched taskbar transparency in newer OS builds
Setup explorer functions done
Setup uxtheme functions done
Setup twinui.pcshell functions done
Setup combase functions done
Setup twinui functions done
Setup stobject functions done
Setup bthprops functions done
Setup pnidui functions done
Setup shell32 functions done
Setup explorerframe functions done
Setup windows.storage functions done
[IME] Context menu patch status: 1
Setup inputswitch functions done
Setup windowsudk.shellcommon functions done
Setup peopleband functions done
Installed hooks.
Play startup sound thread...
Signal shell ready...
Open Start on monitor thread
EP Service Window thread
Started "Signal shell ready" thread.
Started "Open Start on current monitor" thread.
Initialized taskbar centering module.
Progman: 393978
Progman hook: 197035
0 0
Error in sub_1800135EC on RoGetActivationFactory.
Started "Play startup sound" thread.
Ended "Play startup sound" thread.
[Updates] Starting daemon.
